bocas/artifacts/keras_history.py

```diff
@@ -1,16 +1,26 @@
-import pickle
-
 from bocas.artifacts.artifact import Artifact
+from bocas.artifacts.yaml_utils import parse_yaml_node
 
 
 class KerasHistory(Artifact):
     """KerasHistory is used to return the keras `model.fit()` history."""
 
+    yaml_tag = "!KerasHistory"
+
     def __init__(self, history, **kwargs):
         super().__init__(**kwargs)
-        self.history = history.history
+        self.history = history
 
     @property
     def metrics(self):
         """Returns a dictionary holding the metrics from `fit()`."""
         return self.history
+
+    def to_yaml(self):
+        config = super().to_yaml()
+        config.update({"history": self.history.history})
```

Comment:
I think here you need to put just `self.history`, and update `metrics` - right?

Reply:
(unless you update __init__ to take either type

Reply:
Added the update to init to take either type so loading should be good now.